README.md

```diff
@@ -32,6 +32,20 @@ Using this requries no additional dependencies in the client-side code. Just use
     import * as AWS from 'aws-sdk';
     import SageMaker from 'aws-sdk/clients/sagemaker';
 
+    // Generic function to get a cookie value, needed for accessing the XSRF token
+    function getCookie(name: string) {
+      // from tornado docs: http://www.tornadoweb.org/en/stable/guide/security.html
+      var r = document.cookie.match("\\b" + name + "=([^;]*)\\b");
+      return r ? r[1] : undefined;
+    }
+
+    // Reusable function to add the XSRF token header to a request
+    function addXsrfToken<D, E>(request: AWS.Request<D, E>) {
+      const xsrfToken = getCookie('_xsrf');
```

Comment:
Thanks for the contribution!

To keep the README concise, could you condense getCookie into this method itself?

```
const cookie =  document.cookie.match("\\b" + '_xsrf' + "=([^;]*)\\b") 
const xsrfToken = cookie? cookie[1] : undefined
```

Reply:
Sure! Updated in the next commit.